Project/BaseModel.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  BaseModel.swift
+//  Project
+//
+//  Created by Emmanuel Velez Agudelo on 29/10/18.
+//  Copyright © 2018 Wolox. All rights reserved.
+//
+
+import Foundation
+//Generic behavior for all the models
+class BaseModel {
+    
+    
+}
```

Comment:
I don't think you will need this base class, but it won't hurt your app either.

Reply:
ok